Makefile

```diff
@@ -2,14 +2,56 @@
 
 SHELL := /bin/bash
 MAKEFILE_DIR = $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
-PYTHON_VERSION := 3.6
+PYTHON_VERSION := 3.6 # or you can use '3.7'
```

Comment:
how does this interact with the PYTHON variable in Dockerfile.dev?

can you give some comments that this ultimately installs ci/requirements-PYTHON.yml

Reply:
@jreback thanks for review. Comments about PYTHON_VERSION variable were added.